src/test/java/com/training/services/tests/VoyageCurrentSailValidateTest.java

```diff
@@ -43,4 +44,26 @@ public void testVoyageCurrentSailDateValidate(){
                 .matches("2020[0,1][0-2][0-3][0-9]")).
                 isTrue().as("is not in  the year 2020");
     }
+
+    /**
+     * testVoyageSailDurationValidate:validating sail duration should br less than 10
+     */
+    @Test
+    public void testVoyageSailDurationValidate() {
+        Response voyageResponse =
+                new RestEngine().getResponseGet(map.get("url_base")
+                                +"/en/royal/mobile/v3/ships/al/voyages"
+                         ,headerMap)
+                        .as(Response.class);
```

Comment:
create a function which would reuse this, and you can just call this in your test methods